docs/reference/release-notes/highlights-7.4.0.asciidoc

```diff
@@ -0,0 +1,106 @@
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Cumulative cardinality aggregation
+
+https://www.elastic.co/guide/en/elasticsearch/reference/7.4/search-aggregations-pipeline-cumulative-cardinality-aggregation.html[Cumulative cardinality aggregation] has been introduced as part of our ongoing efforts to provide advanced aggregations. This newly added pipeline aggregation allows users to calculate a net-new total of document occurrences within a given time range.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Snapshot lifecycle management
+
+In 7.4.0, we’re introducing https://www.elastic.co/guide/en/elasticsearch/reference/7.4/getting-started-snapshot-lifecycle-management.html[snapshot lifecycle management], which allows an administrator to define policies (via API or UI) that manage when and how often snapshots are taken. This ensures that appropriate, recent backups are ready if disaster strikes or the data needs to be restored for other reasons. 
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== API key management
+
+Elasticsearch is introducing https://www.elastic.co/guide/en/elastic-stack-overview/7.4/security-privileges.html[new cluster privileges to manage API keys], allowing cluster administrators to manage everything, and regular users to manage their own keys. Users can create API keys and use them to provide long-term credentials while interacting with Elasticsearch. 
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== TLS settings for email notifications
+
+Notifications may contain sensitive information that must be protected over the wire. This requires that communication with the mail server is encrypted and authenticated properly.
+In 7.4.0, Elasticsearch supports custom https://www.elastic.co/guide/en/elasticsearch/reference/7.4/notification-settings.html#ssl-notification-smtp-settings[TLS settings for email notifications], allowing secure connections to servers with custom security configuration.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Automatic query cancellation
+
+With the release of version 7.4.0 automatic query cancellation will (automatically) terminate queries sent through the _search endpoint when the initiating connection is closed.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Added support for AdoptOpenJDK
+
+AdoptOpenJDK 13 is now supported and will ship with Elasticsearch as the pre-bundled JDK (as always, if you want to bring your own JDK, you can still do so by setting `JAVA_HOME` before starting Elasticsearch.) The availabiliity of a notarized AdoptOpenJDK package (per the new requirements for software running on macOS Catalina) will faciltate notarization of Elasticsearch for continued support on macOS. 
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Regression analysis - Experimental
```

Comment:
```suggestion
==== Regression analysis
```

It looks like we haven't noted experimental features in the headings of previous release highlights so I removed them here.

Reply:
I actually think we should note that things are experimental here to set expectations with the user and avoid surprise

Reply:
That works for me.